src/main/scala/com/mozilla/telemetry/streaming/ErrorAggregator.scala

```diff
@@ -128,12 +144,26 @@ object Aggregator {
 
   def main(args: Array[String]): Unit = {
     val ssc = new StreamingContext(conf, Seconds(300))
+    val opts = new Opts(args)
+
+    val kafkaParams = Map(
+      "metadata.broker.list" -> opts.kafkaBroker(),
+      "group.id" -> opts.kafkaGroupId(),
+      "auto.offset.reset" -> "largest"
+    )
+
     val stream = KafkaUtils.createDirectStream[String, Message, StringDecoder, HekaMessageDecoder](
       ssc,
       kafkaParams,
       Set("telemetry"))
 
-    process("/tmp/parquet")(stream)  // TODO: parametrize the job
+
+    val prefix = s"error_aggregates/v1"
+    val outputBucket = opts.outputBucket()
+
+    val path = if(opts.saveLocally()) "/tmp/parquet" else s"s3://${outputBucket}/${prefix}/"
```

Comment:
What about using an option to pass a prefix for the output path? In that way we don't have to distinguish in the code if the path is local or not.